### PR TITLE
open-cluster-management/backlog#11934--errata

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

**Review by juliana-hsu — COMMENTED** [review]
Some comments about the updates:
- Small typo - added 's' to Cluster
- 10928 doesn't describe the change accurately, it should say 'Update Red Hat OpenShift release ClusterImageSets available when creating a new cluster'.  The change made was to remove the old release versions that are no longer supported.

Fixed the _Clusters_ page to display imported cluster distributor version and nodes. (GitHub 11776)
Updated Red Hat OpenShift release so that `ClusterImageSets` are available when creating a new cluster. (GitHub 10928)
Fixed an issue with bare metal assets reporting incorrect status. (GitHub 10009)

File: release_notes/fixpack.adoc [hunk 1]
```diff
@@ -5,6 +5,16 @@ By default, Errata updates are automatically applied. See link:../install/upgrad
 
 *Important:* For reference, link:https://access.redhat.com/errata/#/[Errata] links and GitHub numbers might be added to the content and used internally. Links that require access might not be available for the user. 
 
+== Errata 2.0.10
+
+View a summarized list of {product-title} Errata 2.0.10 updates:
+
+. Fixed the _Clusters_ page to display the imported cluster distributor version and nodes. (GitHub 11776)
+
+. Updated Red Hat OpenShift release `ClusterImageSets` that are available when creating a new cluster. (GitHub 10928)
```
Comment (dockerymick) [inline release_notes/fixpack.adoc]: I made a couple of commits already. I suggest changing  `when creating a new cluster` to `when you create a new cluster`. Let me know your thoughts

**Review by dockerymick — APPROVED** [review]
One comment and a couple of commits. It looks good tho, thank you Brandi!

File: release_notes/fixpack.adoc (hunk 1 above)
Comment (swopebe) [reply release_notes/fixpack.adoc]: it works

**Pull request merged**